ProtocolHandler.pm

```diff
@@ -10,6 +10,22 @@ use Slim::Utils::Log;
 use Slim::Utils::Misc;
 use Slim::Utils::Prefs;
 
+use List::Util qw(min max first);
```

Comment:
Do you even use any of these imports?

Reply:
Prefs is used to read prefs, Misc could be removed.